addons/custom-zoom/userscript.js

```diff
@@ -0,0 +1,102 @@
+export default async function ({ addon, global, console }) {
+  async function setZoom(e) {
+    //Function to set the custom zoom parameters
+    if (!global.Blockly) return;
+
+    if (e.detail.newUrl.includes("/editor")) {
+      //Set the zoom parameters
+      global.Blockly.getMainWorkspace().options.zoomOptions.maxScale = addon.settings.get("maxZoom") / 100;
+      global.Blockly.getMainWorkspace().options.zoomOptions.minScale = addon.settings.get("minZoom") / 100;
```

Comment:
if you do want to keep those settings, make sure to listen for when they change

Reply:
Any suggestions? I really have no idea how to do that. I have only been coding for about 4 months. Also, I don't think I need to. I does not seem to change at all.

Reply:
something like
```js
addon.tab.settings.addEventListener("change", (e) => {
    console.log(e.detail)
});
```

Reply:
oh, setting change. I thought you meant variable change, which is a much harder thing to do. I can add that, no problem!

Reply:
how about 
```js
addon.tab.settings.addEventListener("change", function () {
    setZoom({ detail: { newUrl: window.location.href } });
}
```
 The same thing I did with urlchange

Reply:
Yeah that should work